shell.nix

```diff
@@ -15,15 +15,25 @@ mkShell {
     rustc
     cargo
     openssl
-    cargo-make
+    (cargo-make.overrideAttrs (old: {
+      patches = (old.patches or []) ++ [
+        # https://github.com/sagiegurari/cargo-make/pull/388
+        (fetchpatch {
+          url = "https://github.com/sagiegurari/cargo-make/commit/e58f3a969661d9723ed00af802aa50388f28f3ba.patch";
+          sha256 = "1mvhlzs37q61jcgnj20w71rpwi02scw0zy8irvbmxpdfq5zy6m63";
+        })
+
+        # Declutters the log-messages from cargo-make a bit. Don't consider
+        # this one stable, so no PR yet.
```

Comment:
Should we merge it anyway?

Reply:
Not yet IMHO. I'll push an improved version of this patch tonight. In case you all are fine with it, we can merge it then, otherwise I'd wait until it gets merged upstream.